front-end/src/App.js

```diff
@@ -23,29 +24,18 @@ class App extends React.Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-          <p>{this.state.apiResponse}</p>
-          <Router>
-            <Switch>
-              // TODO: Add route for root, e.g "/" and render homepage component
-              <Route path="/editor">
-                <Editor />
-              </Route>
-            </Switch>
-          </Router>
-        </header>
+        <p>{this.state.apiResponse}</p>
+        <Router>
+          <Switch>
+            // TODO: Add route for root, e.g "/" and render homepage component
+            <Route path="/editor">
+              <Editor />
+            </Route>
+            <Route path="/dashboard">
+              <Dashboard />
+            </Route>
```

Comment:
This should be root path
```suggestion
            <Route path="/">
              <Dashboard />
            </Route>
```